rust/index/src/fulltext/types.rs

```diff
@@ -161,6 +161,30 @@ impl FullTextIndexWriter {
         Ok(())
     }
 
+    pub fn get_keys_to_prefetch_before_finish(&self) -> Vec<(String, u32)> {
```

Comment:
Is this just a hack to fetch all the blocks? Why expose this this way instead of just letting blockfile support .get_all_blocks or equivalent